### PR TITLE
support protobuf feature

### PR DESCRIPTION
**Pull request by leizhiyuan**

### CODE REVIEW

**Comment by slievrly** [comment]
I will verify the function.

File: core/src/main/java/io/seata/core/rpc/netty/MessageCodecHandler.java [hunk 1]
```diff
@@ -50,57 +55,81 @@
...
+    private static final short FLAG_REQUEST = 0x80;
+    private static final short FLAG_ASYNC = 0x40;
+    private static final short FLAG_HEARTBEAT = 0x20;
+    private static final short FLAG_SEATA_CODEC = 0x10;
     private static final int MAGIC_HALF = -38;
     private static final int NOT_FOUND_INDEX = -1;
 
+    private static Configuration configuration = ConfigurationFactory.getInstance();
+
+    private static String serialize = configuration.getConfig(ConfigurationKeys.SERIALIZE_FOR_RPC);
```
Comment (slievrly) [inline core/src/main/java/io/seata/core/rpc/netty/MessageCodecHandler.java]: should there be a default value that is compatible with the previous configuration file?

**Comment by leizhiyuan** [comment]
image

It seems ok for me in protobuf

**Comment by slievrly** [comment]
image

**Comment by leizhiyuan** [comment]
> image
> GlobalBeginRequest

I will check it

**Comment by slievrly** [comment]
@leizhiyuan It's my fault. I loaded SW when I started it, causing the class to be delegated.
image

**Pull request merged**